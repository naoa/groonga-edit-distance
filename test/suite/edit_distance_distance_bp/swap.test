plugin_register functions/edit_distance

table_create Tags TABLE_PAT_KEY ShortText UInt32

load --table Tags
[
{"_key": "ai"},
{"_key": "aiue"},
{"_key": "a"}
]

select Tags \
--output_columns '_key, edit_distance_bp("aiu", _key)' \
--command_version 2
