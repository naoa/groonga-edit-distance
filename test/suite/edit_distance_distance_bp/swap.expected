plugin_register functions/edit_distance
[[0,0.0,0.0],true]
table_create Tags TABLE_PAT_KEY ShortText UInt32
[[0,0.0,0.0],true]
load --table Tags
[
{"_key": "ai"},
{"_key": "aiue"},
{"_key": "a"}
]
[[0,0.0,0.0],3]
select Tags --output_columns '_key, edit_distance_bp("aiu", _key)' --command_version 2
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        3
      ],
      [
        [
          "_key",
          "ShortText"
        ],
        [
          "edit_distance_bp",
          "Object"
        ]
      ],
      [
        "a",
        2.0
      ],
      [
        "ai",
        1.0
      ],
      [
        "aiue",
        1.0
      ]
    ]
  ]
]
